src/motion/cursor.ts

```diff
@@ -0,0 +1,33 @@
+import * as vscode from 'vscode';
+
+export default class Cursor {
+	editor: vscode.TextEditor;
+
+	constructor(editor: vscode.TextEditor) {
+		this.editor = editor;
+	}
+	
+	moveLeft(times = 1) {
+		for (var index = 0; index < times; index++) {
+			vscode.commands.executeCommand('cursorLeft');
```

Comment:
+1

`mode_insert.ts` contains an example of how to calculate and set a new position based on line, char (roughly translates to x/y).